src/lib/providers.ts

```diff
@@ -16,15 +17,20 @@ export const getAvailableProviders = async () => {
           openAIApiKey,
           modelName: 'gpt-3.5-turbo',
           temperature: 0.7,
+
         }),
         'gpt-4': new ChatOpenAI({
           openAIApiKey,
           modelName: 'gpt-4',
           temperature: 0.7,
+          configuration: {
+            baseURL: openAIBaseUrl,
+          },
         }),
         embeddings: new OpenAIEmbeddings({
           openAIApiKey,
           modelName: 'text-embedding-3-large',
+          
```

Comment:
Can you add another endpoint param - embedding baseURL as well as an openAIEmbeddingsAPIKey here as well please. 
 
 This is so we can use another provider for the chat model whilst keeping using openai for the embedding model

Reply:
sure. you are right . 

the another question is where to put baseurl config , user config.toml [API_KEYS] section is fine ?